include/boost/decimal/detail/cmath/impl/asin_impl.hpp

```diff
@@ -14,62 +14,81 @@ namespace boost {
+                                                                     decimal64 {UINT64_C(9319383818485265142), -18,
+                                                                                true},
+                                                                     decimal64 {UINT64_C(1876826158920611297), -17},
+                                                                     decimal64 {UINT64_C(2351630530022519158), -17,
+                                                                                true},
+                                                                     decimal64 {UINT64_C(2046603318375014621), -17},
+                                                                     decimal64 {UINT64_C(1304427904865204196), -17,
+                                                                                true},
+                                                                     decimal64 {UINT64_C(6308794339076719731), -18},
+                                                                     decimal64 {UINT64_C(2333806156857836980), -18,
+                                                                                true},
+                                                                     decimal64 {UINT64_C(6826985955727270693), -19},
+                                                                     decimal64 {UINT64_C(1326415745606167277), -19,
+                                                                                true},
+                                                                     decimal64 {UINT64_C(2747750823768175476), -20},
+                                                                     decimal64 {UINT64_C(2660509753516203115), -20},
+                                                                     decimal64 {UINT64_C(3977122944636320545), -22},
+                                                                     decimal64 {UINT64_C(4461135938842722307), -20},
+                                                                     decimal64 {UINT64_C(1826730778134521645), -24},
+                                                                     decimal64 {UINT64_C(7499992533825458566), -20},
+                                                                     decimal64 {UINT64_C(2034140780525051207), -27},
+                                                                     decimal64 {UINT64_C(1666666666327808185), -19},
+                                                                     decimal64 {UINT64_C(2987315928933390856), -31},
+                                                                     decimal64 {UINT64_C(9999999999999989542), -19},
+                                                                     decimal64 {UINT64_C(6087279793251991118), -37}
+                                                             }};
+};
+
+#if !(defined(__cpp_inline_variables) && __cpp_inline_variables >= 201606L) && (!defined(_MSC_VER) || _MSC_VER != 1900)
```

Comment:
Hi Matt, could we turn the logic into an intuitive PP symbol?

```
#if !(defined(__cpp_inline_variables) && __cpp_inline_variables >= 201606L) && (!defined(_MSC_VER) || _MSC_VER != 1900)
```

Reply:
I'll make it `BOOST_DECIMAL_HAS_INLINE_VARIABLES`. Without this you'll get all kinds of linker errors.

Reply:
Awesome. Nice choice and many thanks.